pquery-run.sh

```diff
@@ -1657,6 +1691,16 @@ pquery_test(){
         echoit "Saving full trial outcome (as SAVE_TRIALS_WITH_CORE_OR_VALGRIND_ONLY=0 and so trials are saved irrespective of whether an issue was detected or not)"
         savetrial
         TRIAL_SAVED=1
+      elif [ ${SAVE_TRIALS_WITH_CORE_OR_VALGRIND_ONLY} -eq 1 ]; then
```

Comment:
behaviour with 2.0 should be the same.  will it impact the existing behaviour?

Reply:
Introduced a new variable which is enabled when pquery3 is used. Behaviour of pquery 2.0 is not changed as part of this integration